src/main/java/Mummy.java

```diff
@@ -0,0 +1,194 @@
+import java.util.Scanner;
+
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.ToDo;
+
+public class Mummy {
+  private static final String LOGO =
+        " __  __\n"
+      + "|  \\/  |_   _ _ __ ___  _ __ ___  _   _\n" 
+      + "| |\\/| | | | | '_ ` _ \\| '_ ` _ \\| | | |\n"
+      + "| |  | | |_| | | | | | | | | | | | |_| |\n"
+      + "|_|  |_|\\__,_|_| |_| |_|_| |_| |_|\\__, |\n"
+      + "                                  |___/ \n";
+
+  private static final Store<Task> STORE = new Store<>("Here are the tasks in your list:");
+
+  private enum Command {
+    BYE, LIST, MARK, UNMARK, TODO, DEADLINE,
+    EVENT, DELETE, UNKNOWN
+  }
+
+  public static void main(String[] args) {
+    Scanner scanner = new Scanner(System.in);
+
+    greet();
+    listen(scanner);
+    scanner.close();
+  }
+
+  private static void greet() {
+    echo("Hello from\n"
+            + LOGO + "\n"
+            + "What can I do for you?\n");
+  }
+
+  private static void listen(Scanner scanner) {
+    while (scanner.hasNextLine()) {
+
+      String input = scanner.nextLine();
+      Command command;
+
+      try {
+        command = Command.valueOf(
+                input.split(" ")[0].toUpperCase());
+      } catch (IllegalArgumentException exception) {
+        command = Command.UNKNOWN;
+      }
+
+      try {
```

Comment:
I suggest adding some additional classes such as Command Class and its child classes of AddCommand, ExitCommand etc. This is because the Mummy class has too many methods in its body that can be separated